src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+const FEMALE_SEX = 'f';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const personPartner = sex === FEMALE_SEX
+    ? 'husband'
+    : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
```

Comment:
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
      
      {age && (
```
Add empty lines between elements to increase code readability. Also, would be great to check if the age is a positive number